### PR TITLE
Re-add faktor solution and readme for assignment 2

### PR DESCRIPTION
**Pull request by cjyovich**
Pushing my solution for Assignment 2 again after merging issue.

### CODE REVIEW

File: problems/kattis/faktor/C++/cjyovich/faktor.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+// Solution to Kattis 'Faktor' Problem.
+// Problem Description: https://open.kattis.com/problems/faktor
+// Courtney Yovich, CSCI 494 - Industry Methods - Assignment 2
+
```
Comment (jailarie) [inline problems/kattis/faktor/C++/cjyovich/faktor.cpp:5]: //Wonderful as always. Good call on adding the link to the problem! -JW

**Review by jailarie — COMMENTED** [review]
+1 Great job! Left me without much to comment on. Nicely done!

**Comment by dzumpano** [comment]
Instructions are clear and code runs perfectly!
Capture

**Comment by dzumpano** [comment]
Instructions are clear and well written and code runs as expected
InkedCapture_LI

**Review by jailarie — APPROVED** [review]
//Wonderful as always. Good call on adding the link to the problem! -JW

**Pull request merged**